examples/guide/main.go

```diff
@@ -30,9 +30,9 @@ func main() {
 		// http://<test-client>:8080/?behavior=dance
 		behavior := r.FormValue("behavior")
 
-		// when client is called with no arguments,
-		// report back with a 200 when ready to run tests
-		if behavior == "" {
+		// when method is HEAD, report back with a 200 when ready to run tests
+		if r.Method == "HEAD" {
+			w.Header().Add("Content-Length", "0")
```

Comment:
FYI this is needed so that certain HTTP clients know not to expect a body and disconnect afterwards.